doc/sources/.static/kivy.js

```diff
@@ -224,5 +224,12 @@ $(function() {
 		var section_title = $('li.toctree-l1.current > a').text();
 		$('div.body h1:eq(0)').prepend(section_title + ' &raquo; ');
 	}
-
+	// hack to add a attrib to all linenos
```

Comment:
Can we clarify why is that needed?

Reply:
the comment ?

Reply:
Nope, the comment is fine, just needs to be a little bit verbosely about it.
As an example (but feel free to change it): "Hack to add an attrib to all linenos, this is needed in order to avoid that certain browsers (Ex: Safari) treat the line-number as a copyable item."

Reply:
ahh i see u mean it should be more descriptive, correct